src/main/java/com/javax0/jscc/Compiler.java

```diff
@@ -26,89 +28,99 @@
  */
 public class Compiler {
 
-	private ClassLoader classLoader = Compiler.class.getClassLoader();
-
-	private String compilerErrorOutput = null;
-
-	/**
-	 * Setting the parent class loader for the {@see ByteClassLoader} used to
-	 * load the compiled byte code. This is needed when there are different
-	 * class loaders in the application and the code compiling the class uses
-	 * different class loader than the code wanting to use the compiled class.
-	 * If no class loader is specified via this setter then the actual class
-	 * loader will be used, which may work for most of the cases.
-	 * 
-	 * @param classLoader
-	 */
-	public void setClassLoader(ClassLoader classLoader) {
-		this.classLoader = classLoader;
-	}
+    private ClassLoader classLoader = Compiler.class.getClassLoader();
```

Comment:
General recommendation: when you make a change in a project do not reformat the code, keep the tabbing. Otherwise the 'diff' will show a lot of lines deleted and reinserted and it is hard to assess the modification.